src/services/conversation.service.ts

```diff
@@ -7,22 +7,14 @@ import { Conversation } from '@entities/conversation.entity';
 export class ConversationsService {
   private readonly conversationRepository = getRepository<Conversation>(Conversation);
 
-  listConversations() {
-    return this.conversationRepository.find();
+  listConversationsByUser(userId: number) {
```

Comment:
I think here you can declare what type of data this function will return and also mark the function as async, since the find method from the repository returns a Promise

Reply:
I forgot the async, thank you.
As a general rule, is it necessary to explicitly add the return type to a function if typescript correctly infers it?

Reply:
It is a good practice and that the reason why Typescript is very useful because you can type what are you send and define what are you expect.